src/passkey_child/passkey_child_common.c

```diff
@@ -838,7 +819,7 @@ get_assert_data(struct passkey_data *data)
     }
 
     print_assert_data(data->key_handle_list[index], data->crypto_challenge,
-                      auth_data, signature, user_id);
+                      auth_data, signature, NULL);
```

Comment:
What is the reason that `print_assert_data` still contains the user_id parameter?

Reply:
This is a leftover from my cleaning. I removed it.